styles/globals.css

```diff
@@ -1,8 +1,14 @@
+@import '../styles/global-styles/fonts.css';
+
+:root {
+  --font-regular: 'Varela Round'
```

Comment:
Let's keep the folders cleaner by declaring variables on their own files. For the fonts here we already have a fonts.css file, so let's create variables there.
When we'll have the other viariables in place we can import them here as well.